app/ui/browser-modern/model/profile.js

```diff
@@ -12,6 +12,10 @@ specific language governing permissions and limitations under the License.
 
 import Immutable from 'immutable';
 
-export default Immutable.Record({
+import { register } from '../util/record-constructors';
+
+const VERSION = 1;
```

Comment:
Should increment this every time we care about migrating the records (and it doesn't work automatically). That is to say: whenever we're changing things substantially (not just adding them) and care enough about not losing the state.